app/src/main/java/com/duckduckgo/app/browser/WebViewRequestInterceptor.kt

```diff
@@ -68,6 +71,23 @@ class WebViewRequestInterceptor(
 
         val url = request.url
 
+        if (shouldLoadAsMobileUrl(request)) {
```

Comment:
I'm having problems understanding this part of the logic.

if `request` comes from `m.twitter.com`, `shouldLoadAsMobileUrl` will return `false`, right? if so, it's confusing. I'm reading `shouldLoadAsMobileUrl("m.twitter.com`)`, and my answer will be yes.

Reply:
I've changed it to `shouldChangeToMobileUrl` which hopefully is more explanatory